aries-site/src/examples/templates/forms/RequiredFieldsExample.js

```diff
@@ -0,0 +1,170 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import {
+  Box,
+  Button,
+  CheckBox,
+  Form,
+  FormField,
+  Header,
+  Select,
+  Text,
+  TextInput,
+  TextArea,
+} from 'grommet';
+import { CircleAlert } from 'grommet-icons';
+
+const superPower = ['Flying', 'Sky Runner', 'Invisibility'];
+const weakness = ['Fire', 'PB & J', 'Kryptonite'];
+
+const RequiredFormField = props => {
+  const { required, label, ...rest } = props;
+  return (
+    <FormField
+      label={required ? `${label}*` : label}
+      required={required}
+      {...rest}
+    />
+  );
+};
+
+RequiredFormField.propTypes = {
+  required: PropTypes.bool,
+  label: PropTypes.string,
+};
+
+const FormContainer = ({ ...rest }) => {
+  return (
+    <Box background="background-front" border round="small" overflow="hidden">
+      <Box flex pad={{ horizontal: 'medium', vertical: 'medium' }} {...rest} />
+    </Box>
+  );
+};
+
+const Error = ({ textError, ...rest }) => {
+  return (
+    <Box {...rest} align="center" gap="xsmall" direction="row">
+      <CircleAlert size="small" />
+      <Text size="xsmall">{textError}</Text>
+    </Box>
+  );
+};
```

Comment:
Related to: https://github.com/hpe-design/design-system/pull/999/files#r463792055

```suggestion
const Error = ({ children, ...rest }) => {
  return (
    <Box align="center" background="background-front" gap="xsmall" direction="row"  {...rest} >
      <CircleAlert size="small" />
      <Text size="xsmall">{children}</Text>
    </Box>
  );
};
```